OpenRA.Mods.RA/Traits/Chronoshiftable.cs

```diff
@@ -19,21 +19,29 @@ namespace OpenRA.Mods.RA.Traits
 	[Desc("Can be teleported via Chronoshift power.")]
 	public class ChronoshiftableInfo : ITraitInfo
 	{
+		[Desc("Should the actor die instead of being teleported?")]
 		public readonly bool ExplodeInstead = false;
 		public readonly string ChronoshiftSound = "chrono2.aud";
 
+		[Desc("Should the actor return to its location after the chronoshift weared out?.")]
```

Comment:
Repeating the vocabulary used in the parameter often doesn't really help explain things.